source/auth/tests/mongodb-aws.rst

```diff
@@ -115,3 +116,26 @@ Sample URIs both with and without optional session tokens set are shown below. D
   URI="mongodb://localhost/?authMechanism=MONGODB-AWS"
 
 .. note:: No username, password or session token is passed into the URI. Drivers MUST check the environment variables listed above for these values. If the session token is set Drivers MUST use it.
+
+
+Cached Credentials
+==================
+
+Drivers MUST ensure that they are testing the ability to cache credentials.
+Drivers will need to be able to query and override the cached credentials to
+verify usage.  To determine whether to run the cache tests, the driver can
+check for the absence of the AWS_ACCESS_KEY_ID and of credentials in the URI.
+
+#. Clear the cache.
+#. Create a new client.
+#. Ensure that a ``find`` operation adds credentials to the cache..
+#. Override the cached credentials with an "Expiration" that is within one
+   minute of the current UTC time.
+#. Create a new client.
+#. Ensure that a ``find`` operation updates the credentials in the cache.
+#. Poison the cache with invalid auth content.
```

Comment:
Is it worth describing in more detail how to poison the cache?

Reply:
Yep, done

Reply:
not done?

Reply:
The updated language was `Poison the cache with an invalid access key id.`, it just doesn't show up in this UI.